en/SaveActions.md

```diff
@@ -32,13 +32,13 @@ Add braces encapsulating the complete field content.
 For instance
 
     title = {Test with UPPERCASE word},
-    
+
 gets
 
     title = {{Test with UPPERCASE word}},
 
 However, this procedure is not recommended.
-It is better to use the "Protect Terms" functionaly.
+It is better to use the "Protect Terms" functionally.
```

Comment:
Should be "functionality".

Reply:
New to the topic, so I made such a mistake, I will change.